openedx/core/djangoapps/team_api/tests/test_views.py

```diff
@@ -0,0 +1,437 @@
+"""Tests for the teams API at the HTTP request level."""
+# pylint: disable=maybe-no-member
+import json
+
+from django.conf import settings
+from django.core.urlresolvers import reverse
+from rest_framework.test import APITestCase
+from unittest import skipUnless
+
+from student.tests.factories import UserFactory
+from student.models import CourseEnrollment
+from xmodule.modulestore.tests.factories import CourseFactory
+from .factories import CourseTeamFactory
+from xmodule.modulestore.tests.django_utils import ModuleStoreTestCase
+
+
+@skipUnless(settings.ROOT_URLCONF == 'lms.urls', 'Team APIs are only supported in LMS')
+class TestTeamAPI(APITestCase, ModuleStoreTestCase):
+    """
+    Tests of the Team API
+    """
+
+    test_password = 'password'
+
+    topics_count = 10
+
+    def setUp(self):
+        super(TestTeamAPI, self).setUp()
+
+        teams_configuration = {
+            'topics':
+            [
+                {
+                    'id': 'topic_{}'.format(i),
+                    'team_count': i,
```

Comment:
I don't think the Advanced Setting configuration should include team_count or is_active.

teams_configuration = Dict(
        display_name=_("Teams Configuration"),
        help=_(
            "Enter configuration for the teams feature. Expects two entries: max_team_size and topics, where "
            "topics is a list of topics."
        ),
        scope=Scope.settings
    )